### ISSUE TITLE
Sneezing doesn't force airborne w/ extra range

### ISSUE THREAD

**Issue by ma44** [tgstation/-tg-station#16609]
When I'm playing as virologist (my favorite role) when making a very horrible virus, I always include sneezing, as inside of the code, ingame and wiki says
Bonus
	Forces a spread type of AIRBORNE
	with extra range!
Expect whenever I add sneezing in, there's always a different way of the virus spreading (like contact or blood) and in the code, it doesn't say anything about forcing airborne. 

Possible solution: Set sneezing symptom transmittable to a very high number (like 50) having a transmittable rating of 4 - infinity makes the virus airborne, and that the transmittable value doesn't effect the chance of the virus spreading.

**Comment by optimumtact** [comment]
as we spoke about in irc transmittable seems to affect how well a virus goes through clothing, so that solution wont' work.

Easiest is just to create a new var or hook for symptoms to force set spread rate.

**Comment by RemieRichards** [comment]
It doesn't set airborne, but it forces spread range from the base value of 1, to a base value of 5 (increasing to 6 if it ALSO has airborne)

tl;dr it does still give benefit, just as a range far beyond airborne
